src/main/java/steps/BaseSteps.java

```diff
@@ -54,4 +55,31 @@ public static void goToCart() {
     public static void validateURL(String url) {
         Assert.assertEquals("Guest Checkout URL is not valid", url, getWebDriverInstance().getCurrentUrl());
     }
+
+    public static void validateNewTabURL(String expectedUrl) {
+        var driver = getWebDriverInstance();
+        var windowHandles = driver.getWindowHandles();
+        var newTabHandle = windowHandles.stream()
+                .filter(handle -> !handle.equals(driver.getWindowHandle()))
+                .findFirst()
+                .orElseThrow(() -> new RuntimeException("New tab did not open"));
+        driver.switchTo().window(newTabHandle);
+        var actualUrl = driver.getCurrentUrl();
+        Assert.assertEquals("URL is not valid", expectedUrl, actualUrl);
+    }
+
+    public static void clickOnFacebookButton() {
+        var base = new BasePage();
```

Comment:
Let's keep the variable naming the way classes are named but using camelCase|.
```java
var basePage = new BasePage();
```
Leaving `base` alone can be confusing because there are many `base' classes e.g. BaseTest, BaseSteps, BasteTable, etc.

Reply:
Thank you, corrected in this place and others where i made similar mistake.